### PR TITLE
modify the readme for keyvault certificates sample

### PR DESCRIPTION
**Pull request by zhichengliu12581**
modify the readme for keyvault certificates sample

### CODE REVIEW

**Review by mnriem — COMMENTED** [review]
Why is this being added?

**Comment by lzc-1997-abel** [comment]
HI,@mnriem, I'm not sure why I can't reply to your content. I'll explain it here
The dependency used in starter is optional in azure-security-keyvault-jca, which will cause ClassNoDefException when azure-spring-boot-sample-keyvault-certificates is started.
https://github.com/Azure/azure-sdk-for-java/blob/5af23e5e66a2fb3312a248596771adf26ece1689/sdk/keyvault/azure-security-keyvault-jca/pom.xml#L129-L134

**Comment by mnriem** [comment]
It is an optional dependency in azure-security-keyvault-jca, because it is relocated and shaded in. The dependency should not be added to the starter.

**Comment by chenrujun** [comment]
Hi, @lzc-1997-abel , I checked jar file. `httpclient5` will be shaded in even though it is an optional dependency.

Refs: 
image

>cause ClassNoDefException when azure-spring-boot-sample-keyvault-certificates is started.

Could you please share the log of stack trace?

**Comment by lzc-1997-abel** [comment]
My fault, there is no problem starting the project through maven, it may be my idea configuration is wrong, I will modify the readme and fix this pr.

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-certificates/README.md [hunk 1]
```diff
@@ -1,13 +1,26 @@
 # Azure Spring Boot Key Vault Certificates client library for Java
+
 ## Key concepts
+This sample illustrates how to use [Azure Spring Boot Starter Key Vault Certificates ](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-certificates/README.md:4]: Please move link to `<!-- LINKS -->` at the end of this file.

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-certificates/README.md [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
+
+In this sample, a certificate named `self-signed` is stored into an Azure Key Vault, and a sample Spring application will use its value as a configuration property value.
+
 ## Getting started
 
 ### Environment checklist
 We need to ensure that this [environment checklist][ready-to-run-checklist] is completed before the run.
 
+### Store Certificate
+We need to store certificate `self-signed` into Azure Key Vault with the following steps: [store certificate ](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md#creating-an-azure-key-vault)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-certificates/README.md:14]: Same here.

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-certificates/README.md [hunk 3]
```diff
@@ -1,13 +1,26 @@
...
+
 ## Examples
 ## Troubleshooting
 ## Next steps
+## Run with Maven
+```
+cd azure-spring-boot-samples/azure-spring-boot-sample-keyvault-certificates
+mvn spring-boot:run
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-certificates/README.md:22]: Good point. 👍

**Review by chenrujun — CHANGES_REQUESTED** [review]

**Review by chenrujun — APPROVED** [review]

**Pull request merged**